.env.example

```diff
@@ -21,3 +21,7 @@ DGRAPH_URL=http://ec2-3-236-210-156.compute-1.amazonaws.com:8080
 TELOS_ENDPOINTS=https://testnet.telos.caleos.io
 
 TLOSTO_SEEDS=tlosto.seeds
+
+IPFS_URL=ipfs.infura.io
+IPFS_PROJECT_ID=20mUXy1Rao9DE32ei9mWy9TlpYM
+IPFS_PROJECT_SECRET=3a2b32602886aca0389a2b6294cf65a4
```

Comment:
Probably don't want this committed... but it's too late now. Probably safest bet is to change this SECRET key and then you can leave this one here (once its essentially the wrong key)

Reply:
I got it, I will take a look at it